### ISSUE TITLE
The value of a field will be missing, if the value in the database starts with "[]"

### ISSUE THREAD

**Issue by shenhj2016** [jprante/elasticsearch-jdbc#959]
The value is "[143]abcd" in the database before using elasticsearch-jdbc.This value will be "[143]" in the elasticsearch.

**Comment by shenhj2016** [comment]
There's a good chance it's about PlainKeyValueStreamListener.class

**Comment by jprante** [comment]
Yes, this is a bug. The bracket notation is used internally to generate JSON objects from table rows.
